src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/imports/UnusedImportsCheck.xml

```diff
@@ -29,6 +29,19 @@
  comment {@code {@link List}}. The alternative to avoid introducing a compile-time
  dependency would be to write the Javadoc comment as {@code {&amp;#64;link java.util.List}}.
  &lt;/li&gt;
+ &lt;li&gt;
+ A static method is imported when used as method reference. In that case,
+ only the type needs to be imported and that's enough to resolve the method.
+ &lt;pre&gt;
+ import static java.lang.String.format; // This is not needed
+ class FooBar {
+   public static void main(String[] args) {
+     Optional&amp;lt;String&amp;gt; test = Optional.empty();
```

Comment:
@romani @nrmancuso This seems confusing. As seen just a few lines above, `&lt;` is valid and acceptable here in XML. This look like we are double sanitizing, but I guess it depends on how this file is used.

Is this suppose to be a print out of Javadoc or an exact replica of the Javadoc's source?

Reply:
@mahfouz72 , please test if we have to escape here.

`// This is not needed` --> `// violation, This is not needed`, 
but hold on https://github.com/checkstyle/checkstyle/blob/d35128bb545aa740836c49c45245d504d7f72076/src/main/resources/com/puppycrawl/tools/checkstyle/checks/imports/messages.properties#L19
this import is required to make it compile-able.
It is not optimal, may be, but message should be different.
existing message claim simple "remove this line", new violation should claim that import should be optimized.

Reply:
@mahfouz72 , ping

Reply:
> we already found that it works not very good with our metadata generation and complicates stuff


@romani I am not familiar with this generation stuff but Yes I am removing the code sample from the description it is not needed we have examples to show this

Reply:
> we already found that it works not very good with our metadata generation and complicates stuff


@romani I am not familiar with this generation stuff but Yes I am removing the code sample from the description it is not needed we have examples to show this

sorry for the late reply didn't notice this